### ISSUE TITLE
The Darkness or Condemned 2 (request, no native PC version available)

### ISSUE THREAD

**Issue by Joris-truly** [marinesciencedude/xenia-canary-mousehook#59]
I really appreciate the hook and think it’s very valuable for games that were never released natively on PC and are still stuck on consoles.

Both **The Darkness (2007)** and **Condemned 2 (2008)** run excellent on Xenia and have prior games from the studio that were released on PC natively using the same core engine tools (Riddick: Butcher Bay and Condemned 1). Not sure if this could help in finding the camera hook.

Appreciate it!

**Comment by Clippy95** [comment]
I looked into both of those games some time ago, the only write-able to camera angles were in a row of 4, someone told me it was possibly in quaternion haven't looked more into it other than that

**Comment by Muthaphuckka** [comment]
Definitely would love to see support for The Darkness, game deserves it big time.

**Comment by johhnyjefferson34133** [comment]
I have to agree with [Muthaphuckka](https://github.com/Muthaphuckka) (excellent nickname by the way), these two would be amazing to experience with PC controls.

**Comment by AndylgTom** [comment]
> Я изучал обе эти игры некоторое время назад, единственные ракурсы камеры, которые можно было записать, были в ряд из 4, кто-то сказал мне, что это, возможно, в кватернионе, больше я этим не интересовался.

Is there a guide on how to make a mousehook (or maybe a video guide) for games?

**Comment by Blackgrowl** [comment]
I second this, I absolutely want it for Condemned 2, a very underrated game that is sadly exclusive to Xbox 360.
